Change development environment to Gradle and Android Studio

miguelcardo opened the pull request:
So that it can be built using command-line tools. Plus Android Studio is now the Google-recommended development environment for Android, and runs faster than Eclipse.

ghost commented:
@miguelcardo I think we should also rename the app and repository when at it. Transceive is no longer part of it.

miguelcardo commented:
@petter-fidesmo  Indeed. Best time to do it.

miguelcardo committed:
Move build structure to Gradle
- New directory structure and Gradle configuration
- Test builds using commandline and Android Studio IDE
- Rewrite build instructions in README.md

miguelcardo commented:
@petter-fidesmo @slomo Ready for review!
I'll rename the repo once this is merged

slomo commented on README.md:
```diff
@@ -1,25 +1,40 @@
----------
-![Screenshot showing the two buttons and an example UUID](/docs/Example_screenshot.png "Screenshot showing the two buttons and an example UUID")
+- In the project's root directory, type ``./gradlew build``
+- Copy the Android app's APK from ``build/apk/android-transceive-demo-release-unsigned.apk`` to your phone and install it manually.
```
I installed with gradel via `./gradlew installDebug` I also tried `adb install  build/apk/android-transceive-demo-release-unsigned.apk` but it failed with `Failure [INSTALL_PARSE_FAILED_NO_CERTIFICATES]`

slomo commented:
Works for me so :+1:
